micronaut-langchain4j-core/src/main/java/io/micronaut/langchain4j/store/memory/chat/inmemory/InMemoryChatMemoryConfigurationProperties.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2017-2025 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.langchain4j.store.memory.chat.inmemory;
+
+import io.micronaut.context.annotation.ConfigurationProperties;
+import io.micronaut.core.annotation.Internal;
+
+/**
+ * {@link ConfigurationProperties} implementation for {@link InMemoryChatMemoryConfiguration}.
+ */
+@Internal
+@ConfigurationProperties(InMemoryChatMemoryConfigurationProperties.PREFIX)
+public class InMemoryChatMemoryConfigurationProperties implements InMemoryChatMemoryConfiguration {
```

Comment:
if this is `@Internal` it doesn't need to be public

Reply:
[I removed the `public`modifier](https://github.com/micronaut-projects/micronaut-langchain4j/pull/173/commits/f84e16364449162c9de5ec580f549bacb9010222)